bluesky/broker_callbacks.py

```diff
@@ -25,9 +20,10 @@ class LiveImage(CallbackBase):
     relevant commit is a951b7.
     """
     def __init__(self, field):
+        from matplotlib.pyplot import figure
```

Comment:
I was mistaken, matplotlib.pyplot can be imported without DISPLAY.  Fixed in 807a52b783d9bb7998780be5b33ed7cdcb9bc389.